.github/workflows/file-ultils.yml

```
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: File Utils
+on:
+  pull_request:
+    paths:
+      - 'src/file-utils/**'
+      - '!src/file-utils/**/*.md'
+  push:
+    paths:
+      - 'src/file-utils/**'
+      - '!src/file-utils/**/*.md'
+jobs:
+  test-file-utils:
+    name: Run unit, style, and type checks
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+
+      - name: Install Python 3.8
+        uses: actions/setup-python@3105fb18c05ddd93efea5f9e0bef7a03a6e9e7df
+        with:
+          python-version: '3.8.x'
+
+      - name: Install Poetry
+        uses: Gr1N/setup-poetry@462ac83c852d49e282a1233c4c24c5411696e7c7
+
+      - name: Install Python Dependencies
+        run: python ./eng/build.py install ../src/file-utils
+
+      - name: Cache Dependencies
+        uses: actions/cache@0781355a23dac32fd3bac414512f4b903437991a
+        with:
+          path: ~/.cache/pypoetry/virtualenvs
+          key: ${{ runner.os }}-poetry-${{ hashFiles('poetry.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-poetry-
+
+      - name: Run Tests with Coverage
+        run: python ./eng/build.py coverage:html ../src/file-utils
+
+      - name: Archive code coverage results
+        uses: actions/upload-artifact@e448a9b857ee2131e752b06002bf0e093c65e571
+        with:
+          name: code-coverage-report
+          path: src/file-utils/htmlcov/
+
+      - name: Type Check
+        run: python ./eng/build.py typecheck:xml ../src/file-utils
```

```suggestion
        run: python ./eng/build.py typecheck:xml file-utils
```

The script automatically looks in the `src` directory. So the only time you need to specify directory is when the package is _not_ in `src`.